src/scripts/main.js

```diff
@@ -2,7 +2,27 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tableBody = document.querySelector('tbody');
 
-// write your code here
+tableBody.insertAdjacentHTML('beforeend', `
+  ${people.map((
+    {
+      // eslint-disable-next-line no-shadow
+      name,
```

Comment:
```suggestion
      name: personName,
```
You can rename it how you want in order not to disable eslint
https://wesbos.com/destructuring-renaming